parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -227,7 +227,15 @@ write_files:
     - name: localcluster
       cluster:
         certificate-authority: /etc/kubernetes/certs/ca.crt
+        {{if IsAzureStackCloud}}
```

Comment:
Should we not needed to change dcosmasterresources.t file to make publicIP Allocation as static. https://github.com/msazurestackworkloads/aks-engine/pull/2/files

Reply:
DC/OS support is deprecated, so we really want to leave those files frozen as-is unless there's a compelling reason.

Reply:
in upstream, it is already static right now.